expyfun/stimuli/_tracker.py

```diff
@@ -738,6 +739,16 @@ def __init__(self, trackers, max_lag=1, rand=None):
         self._tracker_history = np.array([], dtype=int)
         self._response_history = np.array([], dtype=int)
         self._x_history = np.array([], dtype=float)
+        
+        self._dealer_id = id(self)
+        self._callback('dealer_identity', json.dumps(dict(
```

Comment:
Check Coveralls, but with callbacks added here, you might need to add tests. Not sure.